server/lib/activities.js

```diff
@@ -16,9 +16,12 @@ module.exports = {
     var interval = '';
     var recurringAmount = null;
     var currency = '';
-    var tags = [];
     var description = '';
     var eventType = '';
+    var provider = '';
+    var caUsername = '';
+    var caLink = '';
```

Comment:
what does `ca` stand for? It's ok to use such abbreviations deep inside the code (e.g. in a limited closure), otherwise it's better to be more explicit (also with autocomplete it's not a big issue), e.g. `ConnectedAccountUsername`